.vortex/installer/src/Prompts/Handlers/Tools.php

```
@@ -298,8 +298,10 @@ public static function getToolDefinitions(string $filter = 'all'): array {
       self::ESLINT => [
         'title' => 'ESLint',
         'present' => fn(): mixed => File::contains($this->destinationDir . '/package.json', '"eslint":') ||
-          File::exists($this->destinationDir . '/.eslintrc.json'),
+          File::exists($this->destinationDir . '/eslint.config.js'),
         'package.json' => function (JsonManipulator $pj): void {
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $pj. Configured minimum length is 3. (undefined)

(ShortVariable)